ds-controller-board/device/virtual_test.go

```diff
@@ -10,7 +10,7 @@ import (
 	"testing"
 
 	"github.com/edgexfoundry/device-sdk-go/v2/pkg/models"
-
+	dsModels "github.com/edgexfoundry/device-sdk-go/v2/pkg/models"
```

Comment:
This is the same package as above, just aliased. Remove this one and use the non-aliased one.